src/screens/Order/ShowOrder.res

```diff
@@ -902,13 +876,46 @@ let make = (~id) => {
             {
               title: "More Payment Details",
               renderContent: () => {
-                <OrderInfo
-                  paymentId=id
-                  orderDict={orderData->getDictFromJsonObject}
-                  openRefundModal
-                  isNonRefundConnector={isNonRefundConnector(orderData)}
-                  isMetadata=true
```

Comment:
`isMetadata` prop can be removed from <OrderInfo /> component and conditional renders can be removed